### PR TITLE
Script for reproducing issue 1906

### PR DESCRIPTION
**Pull request by HSavynetska**
Script for reproducing issue [Navigation app must be activated during active embedded navigation and "MixingAudioSupported" = true](https://github.com/smartdevicelink/sdl_core/issues/1906)

This PR is **not ready** for review.
This is **draft** version of the script.

### CODE REVIEW

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(a7db5cc) branch.
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

File: test_scripts/Defects/6_1/1906_Navigation_app_must_be_activated_during_active_embedded_navigation_and_MixingAudioSupported_true.lua [hunk 1]
```diff
@@ -0,0 +1,80 @@
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1906
+--
+-- Description:
+-- Navigation app must be activated during active embedded navigation and "MixingAudioSupported" = true.
+--
+-- Precondition:
+-- 1) MixingAudioSupported" = true at .ini file
+-- 2) SDL and HMI are started.
+-- 3) Navigation app is registered.
+-- 4) Navigation app in LIMITED and NOT_AUDIBLE due to active embedded audio source
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1906_Navigation_app_must_be_activated_during_active_embedded_navigation_and_MixingAudioSupported_true.lua:10]:
```suggestion
-- 4) Navigation app in LIMITED and AUDIBLE due to active embedded audio source
```

File: test_scripts/Defects/6_1/1906_Navigation_app_must_be_activated_during_active_embedded_navigation_and_MixingAudioSupported_true.lua [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        end)
+      end)
+  end)
+end
+
+local function onEventChange(self)
+  self.hmiConnection:SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true })
+  self.mobileSession1:ExpectNotification("OnHMIStatus",
+    { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE" })
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1906_Navigation_app_must_be_activated_during_active_embedded_navigation_and_MixingAudioSupported_true.lua:58]:
```suggestion
    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" })
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
Closing in favor of #2452

**Pull request closed without merging**